src/randomness/Randomness.sol

```diff
@@ -0,0 +1,65 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.24;
+
+import {BLS} from "../libraries/BLS.sol";
+import {FeistelShuffleOptimised} from "./FeistelShuffleOptimised.sol";
+import {IRandomnessSender} from "../interfaces/IRandomnessSender.sol";
+import {ISignatureSender} from "../interfaces/ISignatureSender.sol";
+import {RandomnessSender} from "./RandomnessSender.sol";
+import {TypesLib} from "../libraries/TypesLib.sol";
+import {SignatureSender} from "../signature-requests/SignatureSender.sol";
+
+library Randomness {
+    bytes public constant DST = bytes("BLS_SIG_BN254G1_XMD:KECCAK-256_SVDW_RO_NUL_");
```

Comment:
might be worth detailing this is the _signing_ DST and not the _randomness_ DST, as I nearly got confused when I saw the `RandomnessSender` cos it's been a while :p